extensions/commands/art/cmd_build_info.py

```diff
@@ -14,6 +15,8 @@
 from conans.model.recipe_ref import RecipeReference
 from conan import conan_version
 
+SERVERS_FILENAME = ".art-servers"
```

Comment:
I'd try to use another name like `SERVERS_FILE_EXTENSION` or ` SERVERS_FILE_EXT` to simplify...

Furthermore, I saw you're getting the path to this `servers` file, so I think this could go as a global variable:

```python
SERVERS_FILE_PATH = os.path.join(os.path.dirname(__file__), ".art-servers")
```
(even dropping `SERVERS_FILE_EXTENSION` if it's only used here)

Reply:
it is not only the file extension but the name of the actual file (it is an encoded file)